### DISCUSSION TITLE
NINA controlled camera not working in MW4

### DISCUSSION THREAD

**Discussion by michael-jon** [mworion/MountWizzard4#362] [Q&A] [answered]
In Dec 2023, a user asked a question about their NINA controlled camera not working in MW4. When I start MW4 I am always getting a Red tab for the camera with NINA already started and the camera (QHY268M) connected. I have NINA 3.1 and I do not see a setting for server enabled in Options - > Advanced as you show in your answer. Any help would be appreciated.

**Comment by mworion** [comment, marked as answer]
Hi michel-jon,in NINA 3x they decided to restructure their software. The remote access is now available in a separate plugin (SGP server emulator), which you have to install an enable.MichelAm 06.08.2024 um 00:08 schrieb michael-jon ***@***.***>:﻿
In Dec 2023, a user asked a question about their NINA controlled camera not working in MW4. When I start MW4 I am always getting a Red tab for the camera with NINA already started and the camera (QHY268M) connected. I have NINA 3.1 and I do not see a setting for server enabled in Options - > Advanced as you show in your answer. Any help would be appreciated.
